kde/frameworks/extra-cmake-modules/extra-cmake-modules.py

```diff
@@ -5,9 +5,13 @@ class subinfo(info.infoclass):
     def setTargets(self):
         self.versionInfo.setDefaultValues()
 
+        for ver in self.versionInfo.tarballs() + self.versionInfo.branches() + self.versionInfo.tags():
+            ecmVer = CraftVersion(ver)
```

Comment:
not used

Reply:
ah oops. Should I use [ver] and get rid of this var or use this var?

Reply:
yes the craft version in qtbase is only used for the version check